rspack.config.ts

```diff
@@ -293,10 +293,32 @@ const appConfig: Configuration = {
   },
   plugins: [
     /**
-     * Adds build time measurement instrumentation, which will be reported back
-     * to sentry
+     * Without this, webpack will chunk the locales but attempt to load them all
+     * eagerly.
      */
-    // new SentryInstrumentation(),
```

Comment:
If this was removed, we should remove the module in `build-utils` as well

Reply:
its still in webpack, can do later